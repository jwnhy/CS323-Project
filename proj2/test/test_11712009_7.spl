struct c {

};
int main(int a, int b) {
    struct c f;
    return f + a;
}