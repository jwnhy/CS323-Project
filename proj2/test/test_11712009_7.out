Error type 7 at Line 5: Derived type can not be used in arithemetic operator.
Error type 8 at Line 5: Invalid return type.
